### PR TITLE
Add Pool Share, Slippage Paid, and Deposit Amount to Add Liquidity form

### PR DESCRIPTION
**Pull request by DannyDelott**
Used to just say "LP Shares", this now shows you a bit more information about your Add Liquidity position.

image

### CODE REVIEW

File: apps/hyperdrive-trading/src/ui/markets/TransactionsAndFaqTabs/TransactionsAndFaqTabs.tsx [hunk 1]
```diff
@@ -11,24 +11,31 @@ export function TransactionAndFaqTabs({
...
+        tabs={[
+          {
+            id: "FAQ",
+            label: "FAQ",
+            onClick: () => {},
+            content: <FAQEntries />,
+          },
+        ]}
+      />
+    </>
```
Comment (ryangoree) [inline apps/hyperdrive-trading/src/ui/markets/TransactionsAndFaqTabs/TransactionsAndFaqTabs.tsx:39]: Why split them?

File: apps/hyperdrive-trading/tailwind.config.js [hunk 2]
```diff
@@ -3,6 +3,22 @@ module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx,css}", "index.html"],
   theme: {
     colors: {
+      aquamarine: {
+        100: "#E8FFF7",
+        DEFAULT: "#15FFAB",
+        600: "#11cc89",
+      },
+      skyBlue: {
```
Comment (ryangoree) [inline apps/hyperdrive-trading/tailwind.config.js:11]: nit:
```suggestion
      "sky-blue": {
```

File: apps/hyperdrive-trading/src/ui/markets/TransactionsAndFaqTabs/TransactionsAndFaqTabs.tsx (hunk 1 above)
Comment (DannyDelott) [reply apps/hyperdrive-trading/src/ui/markets/TransactionsAndFaqTabs/TransactionsAndFaqTabs.tsx:39]: They aren't semantically related like Longs, Shorts, and LP so I think grouping them is weird. It's also nicer to just scroll down a few more pixels than have to click to open the FAQ

File: apps/hyperdrive-trading/index.html [hunk 3]
```diff
@@ -7,7 +7,95 @@
...
+          y1="260.349"
+          x2="1440"
+          y2="260.349"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stop-color="#15FFAB" />
+          <stop offset="1" stop-color="#14D3F9" />
+        </linearGradient>
+      </defs>
+    </svg>
```
[the remark this answers is not in the document]
Comment (DannyDelott) [reply apps/hyperdrive-trading/index.html:98]: Added the background image from Figma:

image

**Pull request merged**